### ISSUE TITLE
Kernel die with Tensorflow and traitlets

### ISSUE THREAD

**Issue by henaff** [microsoft/vscode-jupyter#9321]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

Dying at normalization cell

``` python3
# Here, you will standardize values to be in the [0, 1] range by using tf.keras.layers.Rescaling:
normalization_layer = tf.keras.layers.Rescaling(1./255)

# There are two ways to use this layer. You can apply it to the dataset by calling Dataset.map:
normalized_ds = train_ds.map(lambda x, y: (normalization_layer(x), y))
image_batch, label_batch = next(iter(normalized_ds))
first_image = image_batch[0]

# Notice the pixel values are now in `[0,1]`.
print(np.min(first_image), np.max(first_image))
```

### VS Code Version

1.65

### Jupyter Extension Version

v2022.2.1030672458

### Jupyter logs

```shell
Info 11:07:50: Cell 2 executed with state Success
Info 11:07:50: Cell 4 executed with state Success
Info 11:07:50: Cell 6 executed with state Success
Info 11:07:53: Cell 8 executed with state Success
Info 11:07:53: Cell 10 executed with state Success
Error 11:07:57: Disposing session as kernel process died ExitCode: undefined, Reason: /home/desktop/Desktop/Epitech/T-DEV-810-group45/.venv/lib/python3.8/site-packages/traitlets/traitlets.py:2202: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use 'hmac-sha256' instead of '"hmac-sha256"' if you require traitlets >=5.
  warn(
/home/desktop/Desktop/Epitech/T-DEV-810-group45/.venv/lib/python3.8/site-packages/traitlets/traitlets.py:2157: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use 'ca376549-fc7d-46f8-8e30-14c48b2b5ea8' instead of 'b"ca376549-fc7d-46f8-8e30-14c48b2b5ea8"'.
  warn(

Info 11:07:57: Dispose Kernel process
Error 11:07:57: Raw kernel process exited code: undefined
Info 11:07:57: Cancel all remaining cells due to dead kernel
Info 11:07:57: Cancel pending cells
Info 11:07:57: Cell 12 executed with state Success
```

### Coding Language and Runtime Version

python 3.8.10

### Language Extension Version (if applicable)

v2022.2.1924087327

### Running Jupyter locally or remotely?

Local

**Comment by IanMatthewHuff** [comment]
@henaff Thanks for filing and sorry about the issue. Could you please pull the full logs from the Jupyter output window when this issues happens? I do see the traitlets warning in the snippet you provided, but that warning is a usual benign warning that often shows up in our logging. I'm not sure if that's the root cause so I'd like to be able to look at the full log.
